charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -545,8 +545,14 @@ admissionWebhooks:
   ## Secret labels
   secretLabels: { }
 
+## Agent workloads list
+agents:
+  - name: cloudwatch-agent
+
 agent:
   name:
+  mode: # represents the mode the cloudwatch-agent will run in (deployment, daemonset or statefulset)
+  replicas: # The total number non-terminated pods targeted by this cloudwatch-agent's deployment or statefulSet.
```

Comment:
Maybe set this to -1 as the default to make it obvious its not applicable by default and anything using it will fail unless overriden.